### PR TITLE
working changes to the anyflights package

### PR DESCRIPTION
**Pull request by denironyx**
Hi @simonpcouch , 

I spent a lot of time going through this package and tried to fix as many errors that were occurring. 
Here are some of my addition: 
-  I Included a link to query the interface to DOT (the flight data), which seems to change pretty often.
- I removed the Rcurl::url.exist as it keeps providing me with an error and included a function that provides an output of TRUE if the webserver to the URL is active
- The variable names of some of the data at DOT database have been changed, so I included that changed too.
- I changed the value of the months argument to 1 instead of 1:12. The reason was that it is faster to download and uncompress the dataset for one month as compared to 12 months 

I think the `anyflights` package is a great work and I am more than willing to continue working on this project and get it back on CRAN.

When I was learning R, I spent a lot of time working with the `nycflight13` package which was amazing and wanted to create a package that contains a flight dataset of other cities in America. 

I believe that this package will be very useful to anyone that is currently learning R.

### CODE REVIEW

**Comment by simonpcouch** [comment]
Hi there @denironyx! Thanks for your work on this! I'll take a look at your changes in the next few days and merge if it looks like we can bring this package to a stable state.

I agree that the package could be really useful and appreciate your willingness to contribute—I'll drop some updates here once I get a chance to poke around with the APIs as well.🙂

File: R/get_airlines.R [hunk 1]
```diff
@@ -16,12 +16,12 @@
 #' for weather data, \code{\link{get_airports}} for airport data, and 
 #' \code{\link{anyflights}} for a wrapper function  
 #' @export
-
+source("E:/GITHUB_Projects/anyflights/R/url_check.R")
```
Comment (simonpcouch) [inline R/get_airlines.R:19]: Looks like this `url_check` function is only stored locally. Could you add this function (and relevant documentation if need be) to the `R/` folder?

Comment (denironyx) [reply R/get_airlines.R:19]: Hi @simonpcouch, I just made another push with the `url_check.R` script.

Comment (simonpcouch) [reply R/get_airlines.R:19]: Great, thanks! This `source()` line will still error out on R-CMD-CHECK as it is platform-dependent--the script will be loaded by default on `devtools::load_all()`, so deleting this line should do the trick.

Comment (denironyx) [reply R/get_airlines.R]: I have deleted the `source()` line.

**Comment by simonpcouch** [comment]
Great, thanks so much! Will merge this for now and spend some time here soon getting a sense of what would need to come together for a stable CRAN release.

**Comment by denironyx** [comment]
Hey @simonpcouch, don't hesitate to let me know if you need additional assistance. 

I will do some research as well. Although, I don't have any experience releasing packages on CRAN.

**Pull request merged**